test/action_controller/json_api/linked_test.rb

```diff
@@ -137,6 +142,12 @@ def test_render_resource_with_nested_has_many_include
           assert_equal expected_linked, response['included']
         end
 
+        def test_render_resource_with_include_of_custom_key_by_original
+          get :render_resource_with_include_of_custom_key_by_original
+          response = JSON.parse(@response.body)
+          assert response.key? 'included'
```

Comment:
Sorry, I was half asleep when I posted this. I meant, check that `included` is here and that `hash[:data][:relationships][:reviews]` is here as well.